### ISSUE TITLE
type error '<' not supported between instances of 'str' and 'int'

### ISSUE THREAD

**Issue by binlu1981** [Nextomics/NextPolish#47]
Hi,
It likes that db split process takes long time to run. May I skip split step?
After split step done, genome map step report TypeError which was related align_threads.

  File "/home/usr1/Documents/biosoft/NextPolish/nextPolish", line 144, in map_genome
    THREADS = str(cfg['align_threads']) if cfg['align_threads'] < 5 else '5'
TypeError: '<' not supported between instances of 'str' and 'int'

Any suggestions?

**Comment by moold** [comment]
Yes, see #38 for a solution. and see [here](https://github.com/Nextomics/NextPolish/blob/master/doc/bwa.sh) to skip split step.
